pkg/client/ocm/client.go

```diff
@@ -129,6 +135,14 @@ func (c *client) CreateCluster(cluster *clustersmgmtv1.Cluster) (*clustersmgmtv1
 }
 
 func (c *client) GetOrganisationIdFromExternalId(externalId string) (string, error) {
+	orgId, cached := c.cache.Get(externalId)
+	if cached {
+		orgId, ok := orgId.(string)
+		if ok && orgId != "" {
+			return orgId, nil
+		}
+	}
```

Comment:
can we add a test for when a value is retrieved from cache?

Reply:
It's a bit difficult to add a unit test for this since we can't mock the ams call :disappointed: 

I can look into adding an integration test for this or if it's good enough, have a unit test that sets the cache and ensure it returns correct value without calling to ams. Otherwise, we'd need to move out the cache logic to one of the services to unit test it. wdyt?

Reply:
>  if it's good enough, have a unit test that sets the cache and ensure it returns correct value without calling to ams.

+1,  it is what I was thinking.

Reply:
added the test there :)

Reply:
Thanks